lib/moon/components/switch.ex

```diff
@@ -1,35 +1,11 @@
-defmodule Moon.Components.Switch.Caption do
-  @moduledoc false
-
-  use Moon.StatelessComponent
-
-  prop label, :string
-  prop size, :string, values: ["small", "medium", "large"], default: "medium"
-  prop active, :boolean
-
-  def render(assigns) do
-    ~F"""
-    <span class={
-      "transition-colors duration-200 ease-in-out",
-      "text-moon-9": @size == "small",
-      "text-moon-10": @size == "medium",
-      "text-moon-12": @size == "large",
-      "text-bulma-100": @active,
-      "text-trunks-100": !@active
-    }>
-      {@label}
-    </span>
-    """
-  end
-end
-
 defmodule Moon.Components.Switch do
   @moduledoc false
 
-  use Moon.StatelessComponent
+  use Moon.StatefulComponent
```

Comment:
Why it is stateful now? Value should come from Form Field?

https://github.com/coingaming/moon/blob/main/lib/moon/components/select/multi_select.ex#L144